radis/lbl/calc.py

```diff
@@ -601,6 +607,7 @@ def _calc_spectrum_one_molecule(
     mode,
     export_lines,
     return_factory=False,
+    diluent={},
```

Comment:
don't forget description in Docstrings. 
Also, you can move this parameter just below mole_fractions